### PR TITLE
Add bindings for ScatterChart

### PR DESCRIPTION
**Pull request by tmattio**

### CODE REVIEW

File: src/BsRecharts__Scatter.re [hunk 1]
```diff
@@ -0,0 +1,110 @@
+/* http://recharts.org/en-US/api/Pie */
```
Comment (liubko) [inline src/BsRecharts__Scatter.re]: wrong link

Comment (tmattio) [reply src/BsRecharts__Scatter.re]: Fixed, thanks!

**Comment by liubko** [comment]
@tmattio looks good, thanks for the contribution! merging

**Comment by tmattio** [comment]
Great, thank you :)

**Pull request merged**